docs/UserGuide.md

```diff
@@ -165,29 +182,167 @@ Format: `help`
+              </ul>
+            </td>
+            <td><ul>
+              <li><code>e/jane@gmail.com</code></li>
+              <li><code>e/</code> clears the email field, if there were any and provided the student has other contact data after this deletion.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Parent Phone</td>
+            <td><code>pp/</code></td>
+            <td>Refer to the constraints for <a href="#phone-field">Phone</a> above.</td>
+            <td><ul>
+              <li><code>pp/81234567</code></li>
+              <li><code>pp/</code> clears the parent phone field, if there were any and provided the student has other contact data after this deletion.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Parent Email</td>
+            <td><code>pe/</code></td>
+            <td>Refer to the constraints for <a href="#email-field">Email</a> above.</td>
+            <td><ul>
+              <li><code>pe/john@gmail.com</code></li>
+              <li><code>pe/</code> clears the parent email field, if there were any and provided the student has other contact data after this deletion.</li>
+            </ul></td>
+        </tr>
+        <tr>
+        <td rowspan=5>Optional</td>
+            <td>School</td>
+            <td><code>sch/</code></td>
+            <td>Not Applicable.</td>
+            <td><ul>
+              <li><code>sch/Serangoon JC</code></li>
+              <li><code>sch/</code> clears the school field, if there were any.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Academic Stream</td>
+            <td><code>stream/</code></td>
+            <td>Not Applicable</td>
+            <td><ul>
+              <li><code>stream/A-Level</code></li>
+              <li><code>stream/</code> clears the academic stream field, if there were any.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Academic Level</td>
+            <td><code>lvl/</code></td>
+            <td><ul>
+              <li>Maximum 15 characters, including space(s).</li>
+            </ul></td>
+            <td><ul>
+              <li><code>lvl/J1</code></li>
+              <li><code>lvl/</code> clears the academic level field, if there were any.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Remark</td>
+            <td><code>r/</code></td>
+            <td>Not Applicable.</td>
+            <td><ul>
+              <li><code>r/She is weak at maclaurin series.</code></li>
+              <li><code>r/</code> clears the remark field, if there were any.</li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Tags</td>
+            <td><code>t/</code></td>
+            <td><ul>
+              <li>Must be alphanumeric characters.</li>
+              <li>Case-insensitive</li>
+              <li>To add multiple tags to a student, you need to add <code>t/</code> before every tag name.</li>
```

Comment:
I think adding multiple tags should be address in ladd and not here

Reply:
Just need these examples I think
`t/unpaid` is a valid tag
`t/unpaid retained` is invalid

Reply:
can remove the multiple tags one, since that depends on the command whether it accepts multiple prefixes